native_client/python/client.py

```diff
@@ -62,6 +104,8 @@ def main():
                         help='Print version and exits')
     parser.add_argument('--extended', required=False, action='store_true',
                         help='Output string from extended metadata')
+    parser.add_argument('--json', required=False, action='store_false',
```

Comment:
I missed this, `store_false` makes that a default-enabled parameter.

Reply:
@aayagar001 can you send another pr that switches that to store_true?

Reply:
Sure.

Reply:
Created PR #2546